client/lib/signup/step-actions/index.js

```diff
@@ -450,7 +450,10 @@ export function createAccount(
 				// Fire after a new user registers.
 				recordRegistration( { userData: registrationUserData, flow: flowName, type: 'social' } );
 
-				callback( undefined, pick( response, [ 'username', 'bearer_token' ] ) );
+				callback(
+					undefined,
+					pick( response, [ 'username', 'bearer_token', 'marketing_price_group' ] )
```

Comment:
Does the usersSocialNew endpoint return the marketing_price_group value? Also, like done in the else block, it's safer to assign to a const variable:

> const marketing_price_group = ( response && response.marketing_price_group ) || '';

Reply:
You can simplify tad bit to:
`response?.marketing_price_group ?? ''`

Reply:
Yes, optional chaining for the win. Looks cleaner.

Reply:
Cool tip! Added in d48c50b.